src/Catalog/CatalogWriterBase.cs

```diff
@@ -56,12 +56,12 @@ public void Add(CatalogItem item)
 
             _batch.Add(item);
         }
-        public Task Commit( IGraph commitMetadata, CancellationToken cancellationToken)
+        public Task<IEnumerable<string>> Commit(IGraph commitMetadata, CancellationToken cancellationToken)
```

Comment:
Fixed

---
In reply to: [57124048](https://github.com/NuGet/NuGet.Services.Metadata/pull/60#discussion_r57124048) [](ancestors = 57124048)